prow/github/client.go

```diff
@@ -386,6 +386,24 @@ func (c *Client) CreateStatus(org, repo, ref string, s Status) error {
 	return err
 }
 
+func (c *Client) GetLabels(org, repo string) ([]string, error) {
```

Comment:
Ah, AddLabel creates a new label if it doesn't exist? That's annoying.

You can look at `ListIssueComments` to see how to get all labels.